src/styles/header.scss

```diff
@@ -0,0 +1,80 @@
+$blue-link-color: #00acdc;
+$link-color: #060b35;
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.nav-logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav-bar {
+  margin: 0;
+}
+
+.nav-items {
+  display: flex;
+  list-style: none;
+  height: 60px;
+  align-items: center;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  position: relative;
+  text-align: center;
+  line-height: 60px;
+  margin: 0 10px;
+}
+
+ul li:first-child {
+  margin-left: 0;
+}
+
+ul li:last-child {
+  margin-right: 0;
+}
+
+a {
+  display: inline-block;
+  text-decoration: none;
+  color: $link-color;
+  height: 100%;
+}
+
+a:hover {
+  color: $blue-link-color;
```

Comment:
Ensure that the hover color for links provides enough contrast with the background to be easily readable. This is important for accessibility and user experience.